Sources/SpelledPitch/PitchSpeller/Wetherfield/DirectedGraphScheme.swift

```diff
@@ -16,4 +16,8 @@ struct DirectedGraphScheme <Node>: DirectedGraphSchemeProtocol, UnweightedGraphS
     init (_ contains: @escaping (Edge) -> Bool) {
         self.contains = contains
     }
+    
+    func contains(from start: Node, to end: Node) -> Bool {
```

Comment:
I wonder if `containsEdge(from: Node, to: Node) -> Bool` might flow a little naturally.

Reply:
This may be nicer, I agree. What do you think about `weight(from:to:)`?